app/grandchallenge/reader_studies/views.py

```diff
@@ -1342,47 +1327,41 @@ class DisplaySetUpdate(
         f"{ReaderStudy._meta.app_label}.change_{DisplaySet._meta.model_name}"
     )
     raise_exception = True
+    included_form_classes = (
+        DisplaySetUpdateForm,
+        ComponentInterfaceCreateForm,
+        FileForm,
+    )
+    success_message = format_html(
+        "Display set updated. Image and file import jobs have been queued. "
+        "You will be notified about errors related to image and file imports "
+        "via a <a href={}>notification</a>.",
+        "https://grand-challenge.org/notifications/",
```

Comment:
Lets start to always use the kwargs to `format_html`, these things always end up quite complex.

```suggestion
        "via a <a href={url}>notification</a>.",
        "url=https://grand-challenge.org/notifications/",
```